react-calculator/src/App.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import "./App.css";
+import Button from "./Components/Button";
+
+function App() {
+  const [display, setDisplay] = useState<string>("");
+  const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+  const operators = ["/", "%", "-", "+", "="];
+
+  const clickHandler = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    const value = e.currentTarget.value.toString();
+
+    if (!operators.includes(value)) {
+      console.log(value);
+      setDisplay((currentFormula) => `${currentFormula}${value}`);
+    } else {
+      if (value === "=") {
+        setDisplay(eval(display));
+      } else if (value === "AC") {
+        setDisplay("0");
+      } else {
+        setDisplay((currentFormula) => `${currentFormula}${value}`);
+      }
+    }
```

Comment:
if, else if, else 문이 중첩되어 가독성과 유지보수를 어렵게 만들 수 있을 것 같아요.
if-else 문이 너무 많으면 읽기가 어렵고 조건에 대해 명시적이지 않을 수 있어요. 이런 경우 `Early Return`기법을 활용하면 가독성이 더 나아지도록 리팩터링하실 수 있습니다.

참고로 Early Return이란 조건문에서 먼저 Return할 수 있는 부분은 분리하여 우선 if문 내에서 Return하여 함수를 미리 종료하는 것이에요. 이렇게 하면 뒤의 코드로 진입하지 않아 else나 else-if문을 사용하지 않아도 됩니다.

아래와 같이 하면, 더 잘 읽히지 않나요?ㅎㅎ 😄 
```suggestion
if (!operators.includes(value)) {
  console.log(value);
  setDisplay((currentFormula) => `${currentFormula}${value}`);
  return;
}

if (value === "=") {
  setDisplay(eval(display));
  return;
}

if (value === "AC") {
  setDisplay("0");
  return;
}

setDisplay((currentFormula) => `${currentFormula}${value}`);
```